src/NzbDrone.Core.Test/MusicTests/AlbumRepositoryTests/AlbumRepositoryFixture.cs

```diff
@@ -139,6 +121,26 @@ public void should_not_find_album_in_db_by_incorrect_title(string title)
             album.Should().BeNull();
         }
 
+        [Test]
+        public void should_not_find_album_when_two_albums_have_same_name()
```

Comment:
This works for now as its better than errors, but should we somehow integrate disambiguation in at different places to make this a bit more robust? Something to think about for future.